src/components/popup/balances-and-activity.tsx

```diff
@@ -1,14 +1,20 @@
-import React from 'react';
+import React, { useMemo } from 'react';
 import { Box, Stack, SlideFade } from '@stacks/ui';
+import type { StackProps } from '@stacks/ui';
 import { TokenAssets } from '@components/popup/token-assets';
+import dayjs from 'dayjs';
+import relativeTime from 'dayjs/plugin/relativeTime';
 
-import type { StackProps } from '@stacks/ui';
 import { Caption } from '@components/typography';
 import { NoActivityIllustration } from '@components/vector/no-activity';
 import { Tabs } from '@components/tabs';
 import { useAccountActivity } from '@common/hooks/use-account-activity';
-import { TxItem } from '@components/popup/tx-item';
+
 import { useHomeTabs } from '@common/hooks/use-home-tabs';
+import { createTxDateFormatList } from '@common/group-txs-by-date';
+import { TransactionList } from './transaction-list';
+
+dayjs.extend(relativeTime);
```

Comment:
is this needed here? wonder if it could go here with the others https://github.com/blockstack/stacks-wallet-web/pull/1268/files#diff-453269179bb827ed97d74162b95438b94bee34f36992c7d2ab0eca100f6d3395R10

Reply:
Good spot, it's not actually needed at all now we're doing relative dates for only today and yesterday